### PR TITLE
Issue #20 iman

### PR DESCRIPTION
**Pull request by ImanTanumihardja**
Added left and right margin for the main-info-container-fluid class.
Had to rename the class main-info container-fluid to main-info-container-fluid in order to access the class in CSS.

### CODE REVIEW

**Comment by hannah627** [comment]
Hi Iman! Thank you for your work here, it looks good overall! Main thing would be to please make this a pull request into home-page-dev rather than master (I'm not sure if you can edit your pull request, so if you can't, please delete this one and open a new one into home-page-dev). If you have questions on how to do this, feel free to reach out to me or alejandro on discord, and thank you again!

File: styles.css [hunk 1]
```diff
@@ -11,6 +11,11 @@ body {
   padding-top: 0px;
 }
 
+.main-info-container-fluid {
+  margin-left: 172px;
+  margin-right: 172px;
```
Comment (Mr-Friendly) [inline styles.css:16]: Hey Iman, sorry to get this to you so late but we were looking for 342px on each side, not 342/2px. If you could update this we'll approve right away. 


Thank you!

**Review by Mr-Friendly — CHANGES_REQUESTED** [review]
Update 172px on linex 15 and 16 of styles.css to 342px, then can be approved.

**Review by Mr-Friendly — APPROVED** [review]
Looks good Iman, thank you!

**Pull request merged**